src/api/index.js

```diff
@@ -316,3 +316,20 @@ export const fetchMergePullRequest = (
     })
   );
 };
+
+export const fetchSubmitNewIssue = (
```

Comment:
Hmm, this method's name is a little funky. `submitNewIssue` might make more sense.